gem/lib/earlgrey/cli.rb

```diff
@@ -19,11 +19,12 @@ class CLI < Thor
     package_name 'EarlGrey'
 
     desc 'install', 'Installs EarlGrey into an Xcode unit test target'
+    method_option :carthage, type: :boolean, default: true
     method_option :project, aliases: '-p', type: :string, required: false, desc: 'Project'
-    method_option :target,  aliases: '-t', type: :string, required: true, desc: 'EarlGreyTestTarget'
     method_option :scheme,  aliases: '-s', type: :string, required: false, desc: 'EarlGreyTestTarget.xcscheme'
     method_option :swift,    type: :boolean, default: true
-    method_option :carthage, type: :boolean, default: true
+    method_option :swift_version, type: :numeric, default: 3.0, :enum => %w(2.2, 3.0)
```

Comment:
Numerics in Thor are parsed as floats or integers. This doesn't work for point releases of Swift. For example, the upcoming [Swift `3.0.1`](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160725/025676.html).

The correct datatype is string.